tests/fixtures/1_single_zone_tiny_house.json

```diff
@@ -0,0 +1,961 @@
+                "type": "FaceProperties",
+                "energy": {
+                    "type": "FaceEnergyPropertiesAbridged"
+                }
+            },
+            "face_type": {
+                "type": "Wall"
+            },
+            "boundary_condition": {
+                "type": "Outdoors",
+                "sun_exposure": "SunExposed",
+                "wind_exposure": "WindExposed",
+                "view_factor": "autocalculate"
+            },
+            "parent": "TinyHouseZone"
+        },
+        {
+            "type": "FaceAbridged",
+            "name": "TinyHouseZone_Top",
+            "name_original": "Tiny House Zone_Top",
+            "geometry": {
+                "type": "Face3D",
+                "boundary": [
+                    [
+                        5.0,
+                        0.0,
+                        3.0
+                    ],
+                    [
+                        5.0,
+                        10.0,
+                        3.0
+                    ],
+                    [
+                        0.0,
+                        10.0,
+                        3.0
+                    ],
+                    [
+                        0.0,
+                        0.0,
+                        3.0
+                    ]
+                ]
+            },
+            "properties": {
+                "type": "FaceProperties",
+                "energy": {
+                    "type": "FaceEnergyPropertiesAbridged"
+                }
+            },
+            "face_type": {
+                "type": "RoofCeiling"
+            },
+            "boundary_condition": {
+                "type": "Outdoors",
+                "sun_exposure": "SunExposed",
+                "wind_exposure": "WindExposed",
+                "view_factor": "autocalculate"
+            },
+            "parent": "TinyHouseZone"
+        }
+    ],
+    "shades": [
+        {
+            "type": "ShadeAbridged",
+            "name": "TinyHouseZone_Back_InShd0",
+            "name_original": "Tiny House Zone_Back_InShd0",
+            "geometry": {
+                "type": "Face3D",
+                "boundary": [
+                    [
+                        5.0,
+                        0.0,
+                        2.9999999
+                    ],
+                    [
+                        0.0,
+                        0.0,
+                        2.9999999
+                    ],
+                    [
+                        0.0,
+                        0.5,
+                        2.9999999
+                    ],
+                    [
+                        5.0,
+                        0.5,
+                        2.9999999
+                    ]
+                ]
+            },
+            "properties": {
+                "type": "ShadeProperties",
+                "energy": {
+                    "type": "ShadeEnergyPropertiesAbridged",
+                    "diffuse_reflectance": 0.2,
```

Comment:
@chriswmackey are there any default values for these properties?

Reply:
@tanushree04 , I just revised this since the keys are all optional (the samples now don't have them included).  The default values come from EnergyPlus and are as follows:
`diffuse_reflectance` = 0.2
`specular_reflectance` = 0
`transmittance` = 0
`transmittance_schedule` = None